public/app/features/alerting/unified/components/rule-editor/labels/LabelsFieldInForm.tsx

```diff
@@ -13,6 +14,16 @@ interface LabelsFieldInFormProps {
 export function LabelsFieldInForm({ onEditClick }: LabelsFieldInFormProps) {
   const { watch } = useFormContext<RuleFormValues>();
   const labels = watch('labels');
+  const type = watch('type');
+
+  const isGrafanaRecordingRule = type === RuleFormType.grafanaRecording;
+  const text = isGrafanaRecordingRule
```

Comment:
Is this still relevant for cloud rules?

Reply:
this component is only rendered for grafana-managed whatever

Reply:
Updating also for cloud recording rules!